apps/gnome-mobile/src/app/(app)/(tabs)/_layout.tsx

```diff
@@ -2,10 +2,13 @@ import { Redirect } from "expo-router";
 import { SafeAreaView } from "react-native-safe-area-context";
 import HomeTabs from "@/app/(app)/navigator/HomeTabs";
 import LoadingScreen from "@/components/LoadingScreen";
+import { WelcomeBottomSheet } from "@/components/WelcomeBottomSheet";
 import { useAuthStore } from "@/store/useAuthStore";
+import { useIsFirstTimeStore } from "@/store/useIsFirstTimeStore";
 
 export default function AppLayout() {
   const { accessToken, isLoading } = useAuthStore();
+  const { isFirstTime, setIsFirstTimeToFalse } = useIsFirstTimeStore();
```

Comment:
is first time co?

nazwa mało czytelna

może coś pokroju isFirstAppEntry, to od razu mówi dokładnie co to oznacza